### PR TITLE
fix linkerd perf installation issue and add comparison results

### PR DESCRIPTION
**Pull request by carolynhu**

### CODE REVIEW

File: perf/benchmark/linkerd/setup_linkerd.sh [hunk 1]
```diff
@@ -20,22 +20,27 @@ set -ex
...
 # shellcheck disable=SC2046
 # shellcheck disable=SC2086
 kubectl create clusterrolebinding cluster-admin-binding-$USER \
     --clusterrole=cluster-admin --user=$(gcloud config get-value account)
 
+# To check that your cluster is configured correctly and ready to install the control plane
 linkerd check --pre
 
 # shellcheck disable=SC2086
-linkerd install --linkerd-version=$release --proxy-auto-inject | kubectl apply -f -
```
Comment (howardjohn) [inline perf/benchmark/linkerd/setup_linkerd.sh]: why not auto inject?

File: perf/benchmark/setup_test.sh [hunk 2]
```diff
@@ -47,17 +47,17 @@ function run_test() {
   helm -n "${NAMESPACE}" template \
       --set rbac.enabled="${RBAC_ENABLED}" \
       --set includeOutboundIPRanges=$(svc_ip_range) \
-      --set injectL="${LINKERD_INJECT}" \
+      --set client.injectL="${LINKERD_INJECT}" \
```
Comment (howardjohn) [inline perf/benchmark/setup_test.sh]: What about server?

If we turn on linkerd don't we want to turn off istio injection?

File: perf/benchmark/linkerd/setup_linkerd.sh (hunk 1 above)
Comment (carolynhu) [reply perf/benchmark/linkerd/setup_linkerd.sh]: I followed the official document command, --linkerd-version flag fails.

**Comment by howardjohn** [comment]
if we don't have proxy auto inject then we aren't actually testing with
linkerd it seems like? Unless they made that the default or something

File: perf/benchmark/linkerd/setup_linkerd.sh (hunk 1 above)
Comment (carolynhu) [reply perf/benchmark/linkerd/setup_linkerd.sh]: `--proxy-auto-inject` flag is either deprecated or does not exist

**Comment by carolynhu** [comment]
@howardjohn Linkerd automatically adds the data plane proxy to pods when the` linkerd.io/inject: enabled` annotation is present on a namespace or any workloads such as deployments or pods.

**Pull request merged**